packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -36,6 +38,9 @@ const String kDart2jsOptimization = 'Dart2jsOptimization';
 /// Whether to disable dynamic generation code to satisfy csp policies.
 const String kCspMode = 'cspMode';
 
+/// Base href to set in index.html in flutter build command
+const String kBaseHref = '/';
```

Comment:
This is used both as a map key and as a value. Is it supposed to be a key or a value?

Reply:
changed its value to baseHref